Amazon.Ion.ObjectMapper.Test/TestObjects.cs

```diff
@@ -369,6 +369,38 @@ public override string ToString()
         }
     }
 
+    public class Ruler
+    {
+        public int length { get; set; }
+        
+        [IonField]
```

Comment:
Why do you need an `IonField` in a PR about method getters and setters?

Reply:
Because I want to test the duplicate field name check that happens in `IonObjectSerializer.cs` lines 150-154.